torch/onnx/symbolic_helper.py

```diff
@@ -408,18 +408,40 @@ def _interpolate_warning(interpolate_mode):
                   "We recommend using opset 11 and above for models using this operator.")
 
 def _unsqueeze_helper(g, input, axes_i):
-    if _export_onnx_opset_version >= 13:
-        axes = g.op("Constant", value_t=torch.tensor(axes_i, dtype=torch.long))
-        return g.op("Unsqueeze", input, axes)
-    else:
-        return g.op("Unsqueeze", input, axes_i=axes_i)
+    if _is_constant(axes_i[0]):
+        if _export_onnx_opset_version >= 13:
+            axes = g.op("Constant", value_t=torch.tensor(axes_i, dtype=torch.long))
+            return g.op("Unsqueeze", input, axes)
+        else:
+            return g.op("Unsqueeze", input, axes_i=axes_i)
+    else:  # Tensor type
+        if _export_onnx_opset_version >= 13:
+            axes_t = axes_i[0]
+            return g.op("Unsqueeze", input, axes_t)
+        else:
+            raise ValueError("Unsupported Unsqueeze dynamic axes ONNX opset version: " + str(_export_onnx_opset_version))
 
 def _squeeze_helper(g, input, axes_i):
-    if _export_onnx_opset_version >= 13:
-        axes = g.op("Constant", value_t=torch.tensor(axes_i, dtype=torch.long))
-        return g.op("Squeeze", input, axes)
-    else:
-        return g.op("Squeeze", input, axes_i=axes_i)
+    if _is_constant(axes_i[0]):
+        if _export_onnx_opset_version >= 13:
+            axes = g.op("Constant", value_t=torch.tensor(axes_i, dtype=torch.long))
+            return g.op("Squeeze", input, axes)
+        else:
+            return g.op("Squeeze", input, axes_i=axes_i)
+    else:  # Tensor type
+        if _export_onnx_opset_version >= 13:
+            axes_t = axes_i[0]
+            axes_rank = _get_tensor_rank(axes_t)
+            if axes_rank == 0:
+                # axes is a scalar. Unsqueeze it to a rank 1 tensor.
+                axes_t = _unsqueeze_helper(g, axes_t, [0])
+                return g.op("Squeeze", input, axes_t)
+            elif axes_rank == 1:
+                return g.op("Squeeze", input, axes_t)
+            else:
+                raise ValueError("For squeeze axses as input, the rank must be one in ONNX spec")
+        else:
+            raise ValueError("Unsupported Squeeze dynamic axes ONNX opset version: " + str(_export_onnx_opset_version))
```

Comment:
Make the error message actionable for the user by explaining how they can fix the problem.

```suggestion
            raise ValueError(f"opset version must be >= 13 for Squeeze with dynamic axes.")
```

Reply:
Done.